docs/notebooks/pivoted_document_length_normalisation.ipynb

```diff
@@ -0,0 +1,383 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": false,
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
```

Comment:
Missing introduction part:
- what is it
- related links
- why this can be useful